src/theme/global.scss

```diff
@@ -269,6 +269,25 @@ video {
   cursor: pointer;
 }
 
+.combineButton {
+  display: flex;
+}
+
+.exitToProfile{
+  max-width: 150px;
+  margin-top: 24px;
+ }
+
+.redeemCode {
+  max-width: 200px;
+  margin-top: 24px;
+}
+
+.redeemAnotherCode{
+  max-width: 220px;
+  margin-top: 24px;
+}
```

Comment:
Why are we using 2 different classes? It may be better to use a common class in both places and prescribe a padding for the button instead of a max width.

Also, since these classes are specific to the Redeem Modal, perhaps they should be used in the modular scss file RedeemModal.scss